authors/apps/articles/models.py

```diff
@@ -0,0 +1,22 @@
+from authors.apps.authentication.models import User
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+
+class Article(models.Model):
+    slug = models.CharField(max_length=100, blank=False)
+    title = models.CharField(max_length=300, blank=False)
+    description = models.TextField(blank=False)
+    body = models.TextField(blank=False)
+    tag_list = ArrayField(models.CharField(max_length=200), blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    favorited = models.BooleanField(default=False)
+    favorites_count = models.IntegerField(default=0)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    image_url = models.URLField(blank=False)
+    audio_url = models.URLField(blank=True, null=True)
```

Comment:
what does this audio_url field do ..?

Reply:
One can upload audios with their article